src/main/java/org/fundacionjala/coding/yerel/DnaStrand.java

```diff
@@ -0,0 +1,20 @@
+package org.fundacionjala.coding.yerel;
+
+/**
+ * class replace "A" for "T" and "T" for "A".
+ * replace "G" for "C" and "C" for "G".
+ */
+public class DnaStrand {
+
+    /**
+     * @param dna is a AND to convert.
+     * @return ADN convert.
+     */
+    public String makeComplement(String dna) {
+        String dnaCopy = dna;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "dnaCopy". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)